build.gradle

```diff
@@ -2,23 +2,15 @@
 
 buildscript {
     repositories {
-        google()
         jcenter()
-        
-    }
-    dependencies {
-        classpath 'com.android.tools.build:gradle:3.4.0'
-        
-        // NOTE: Do not  place your application dependencies here; they belong
-        // in the individual module build.gradle files
-        classpath "org.jfrog.buildinfo:build-info-extractor-gradle:4.9.5"
+        mavenCentral()
```

Comment:
move to repositories block above

Reply:
Not sure what you mean by this. I copied this over from the config on another provider.

Reply:
So that typically goes under repositories{}. It is not a dependency.  Which repo did you see that in.  It's possible a mistake slipped through.